src/couch/src/couch_att.erl

```diff
@@ -680,11 +680,24 @@ is_compressible(Type) ->
 
 
 max_attachment_size() ->
-    case config:get("couchdb", "max_attachment_size", "infinity") of
+    max_attachment_size(config:get("couchdb", "max_attachment_size", "infinity")).
```

Comment:
I think we can switch to `config:get_integer("couchdb", "max_attachment_size", ?ATT_SIZE_LIMIT)` to simplify `max_attachment_size` and for consistency with other places.  The only possible type `config:get_integer/3` can return is integer. Which means we can write the following:

```
max_attachment_size() ->
     min(config:get_integer("couchdb", "max_attachment_size", ?ATT_SIZE_LIMIT), ?ATT_SIZE_LIMIT).
```

Reply:
Same here as `max_document_size`, changing the third argument in `config:get/3` breaks the tests.

Reply:
See https://github.com/apache/couchdb/commit/fbd84fe72734a44bc61b9e7c2f9cc95548252fed.